core/templates/dev/head/css/about.css

```diff
@@ -0,0 +1,369 @@
+/*
```

Comment:
@BenHenning -- wanted to get your input here. We're thinking of splitting up the CSS since oppia.css is becoming too big. But this is a new pattern, so, is it better to have an about.css file or to put this CSS inline in pages/about/about.html? If an about.css file where do we put it -- in css/ or in pages/about/ ?

Thanks!

Reply:
I've always had some trouble contemplating how to split up oppia.css. The main issue is we want to encourage reusability, but I get the sense that CSS will be more likely duplicated than the corresponding HTML/JS code that accompany it.

I think a basic model would be to keep the CSS as close to the component using it as possible, similar to what we do with JS services and directives. If some CSS is specific to the about page, it should go into an ``about.css`` file in ``pages/about/``. If any CSS is shareable, it should go into an appropriately named file under ``components`` in the same fashion (next to the directive that uses it). Top-level CSS that's shared across the whole site will likely need to live in ``pages/``, but that's not clear yet.

This is just an initial pattern we could start using and see where it leads us. The longer term effort of splitting up oppia.css may actually be harder than this pattern suggests, and we'll then need to think of something else. Does this seem reasonable?

Reply:
@seanlip: Can you please take a look at this and finalize what we need to do here.  
Thanks!

Reply:
I'm fine with following Ben's approach. Does it sound reasonable to you?

Reply:
Yes, I'll try it out and see how it works.

Reply:
@seanlip and @BenHenning: I had a doubt: we use `oppia-about-anchor` in `terms.html` and `privacy.html`. So it should be kept in `oppia.css` only?  
Thanks!